cmd/helm/template.go

```diff
@@ -194,16 +194,15 @@ func (t *templateCmd) run(cmd *cobra.Command, args []string) error {
 		KubeVersion:   chartutil.DefaultKubeVersion,
 		TillerVersion: tversion.GetVersionProto(),
 	}
-	// If --kube-versionis set, try to parse it as SemVer, and override the
```

Comment:
Hey, it looks like this was accidentally deleted in the last rebase. Would you mind adding this back in?

Reply:
From the first I deleted it intentionally, because --kube-version will begin to have the default value by this change. So I felt this line is not needed, and deleted it.

Reply:
Ah gotcha. I was confused why only half of the comment was being removed. Carry on :)